hermes/src/store/proof/wormhole_merkle.rs

```diff
@@ -108,9 +108,13 @@ pub fn construct_update_data(mut message_states: Vec<&MessageState>) -> Result<V
                         .vaa
                         .clone();
 
+                    vaa.span.in_scope(|| {
+                        tracing::info!("Constructing update data for {} Messages.", messages.len())
+                    });
+
                     Ok(to_vec::<_, byteorder::BE>(&AccumulatorUpdateData::new(
                         Proof::WormholeMerkle {
-                            vaa:     vaa.into(),
+                            vaa:     (*vaa).to_owned().into(),
```

Comment:
Temporary hack, would not be able to store Span's with database storage for example.